### ISSUE TITLE
Ostinato Application Icon broken on OS X

### ISSUE THREAD

**Issue by ehlers** [pstavirs/ostinato#260]
Since I started using Ostinato on OS X (from v0.7/QT4 up to master/QT5) on OS X the icon in the Finder and the Dock shows only the lower-right part of logo.

![logo_finder](https://user-images.githubusercontent.com/896406/39967534-4cc46e3c-56bd-11e8-892b-aa431093d1f0.png)

**Comment by ehlers** [comment]
Oh, I got it. This icon seems to be done intentionally that way. The files about.png logo.icns and logo.ico all look that way. I'm not a fan of the that, but if you like it, it's fine.

Close this issue?

**Comment by ehlers** [comment]
Just a comment:

I fully understand that the normal ostinato icon doesn't look nice, when scaled down to a small size, e.g. 16x16 or 32x32. The current solution is fine for such a low resolution, even though it looks different compared to the real ostinato logo.

But OS X icons are really big, the original icon is 128x128. And I doesn't fit well with the other OS X icons, here a screenshot:
![logo_old](https://user-images.githubusercontent.com/896406/40133374-d1939cb2-593f-11e8-9e77-24da880d3027.png)

As I'm not a graphics designer, I looked what other already existing icons can be used as a base for an alternative. I took the ostinato-3d-icon.svg icon (created for GNS3), stretched the height a bit and converted it to a .icns file. Now I like it better.
![logo_new](https://user-images.githubusercontent.com/896406/40133617-93953b7c-5940-11e8-9212-e57571570d4c.png)
